src/cmd/init.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var path = require('path');
+var async = require('async');
+var program = require('commander');
+var plugins = require('../plugins');
+
+
+// error on unknown commands
+program.on('command:*', function () {
+	console.error('Invalid command: %s\nSee --help for a list of available commands.', program.args.join(' '));
+	process.exit(1);
+});
+
+module.exports = {
+
+	loadPluginCommands: function (next) {
```

Comment:
Can you change these to have the following syntax:

```
function abc(...) {
  ...
}
module.exports.abc = abc;
```

Thanks

Reply:
Ok [#0825375](https://github.com/NodeBB/NodeBB/pull/6901/commits/08253750281eab6b95ed9846299140b42f53396e)